test/coralogix.test.js

```diff
@@ -163,6 +163,28 @@ it('CoralogixLogger', async () => {
       application: 'baz',
       subsystem: 'bang',
     });
+
+    logger.log({
+      level: 'info',
+      timestamp: 'boo',
+      message: ['foo'],
+      host: 'bar',
+      application: 'baz',
+      subsystem: 'bang',
+    });
+
+    const req3 = await server.nextReq();
+    const text3 = JSON.parse(req3.logEntries[0].text);
+    delete text3.timestamp;
+
+    ckEq(text3, {
+      message: 'foo',
+      level: 'info',
+      host: 'bar',
+      application: 'baz',
+      subsystem: 'bang',
+      infrastructure: 'Timestamp passed invalid: boo',
```

Comment:
why not:
```
Error: Invalid timestamp passed: boo
```

Reply:
can you be more specific?
- [ ] `error: 'Invalid timestamp passed: boo'`
- [ ] `infrastructure: 'Error: Invalid timestamp passed: boo'`